assets/js/modules/ads/pax/services.js

```diff
@@ -52,7 +52,20 @@ const restFetchWpPages = async () => {
  * @return {Object} An object containing various service interfaces.
  */
 export function createPaxServices( registry ) {
+	const accessToken =
+		global?._googlesitekitPAXConfig?.authAccess?.oauthTokenAccess?.token;
+
 	return {
+		authenticationService: {
+			// eslint-disable-next-line require-await
+			get: async () => {
+				return { accessToken };
+			},
```

Comment:
I think we can use something like this to eliminate a need to disable the eslint rule:
```suggestion
			get: () => Promise.resolve( { accessToken } ),
```
The same for the fix property.

Reply:
Interesting idea. Although we have same format for all other services, for consistency sake, better to keep it with same approach here as well